assets/js/script.js

```diff
@@ -1,125 +1,239 @@
+        btn.addEventListener('click', el => {
+          let thisEl = el.target;
+          if (!thisEl.classList.contains('active')) {
+            thisEl.classList.add('active');
+            thisEl.innerHTML = thisEl.dataset.hide;
+            thisEl.previousElementSibling.style.display = "block"; // no animation
+          } else {
+            thisEl.classList.remove('active')
+            thisEl.innerHTML = thisEl.dataset.show;
+            thisEl.previousElementSibling.style.display = "none"; // no animation
+          }
+        });
+      });
+    },
+
+    initQuestionToggle: () => {
+      let questionBtns = document.querySelectorAll('.question_js');
+
+      questionBtns.forEach(questionBtn => {
+        questionBtn.addEventListener('click', el => {
+          el.preventDefault();
+          let question = el.target.closest('.question');
+          if (!question.classList.contains('active')) {
+            question.classList.add('active');
+            question.querySelector('.question__inner').style.display = "block"; // no animation
+          } else {
+            question.classList.remove('active');
+            question.querySelector('.question__inner').style.display = "none"; // no animation
+          }
+        });
+      });
+    },
+
+    initOnlineToggle: () => {
+      let onlineToggleBtn = document.querySelector('.online_toggle_js');
+      onlineToggleBtn.addEventListener('click', el => {
+        el.target.closest('.online-block').classList.toggle('active');
+      });
+
+      setTimeout(function () {
+        document.querySelector('.online-block').classList.add('visible');
+        document.querySelector('.online-block').classList.remove('hidden');
+      }, 2000);
+    },
+
+    initComments: () => {
+      let comments = document.querySelectorAll('.comment_js');
+
+      comments.forEach(comment => {
+        comment.addEventListener('click', el => {
+          let thisEl = el.target.closest('.comment_js');
+          if (thisEl.classList.contains('active')) return false;
+
+          var activeEl = document.querySelector(".comment_js.active");
+          activeEl.classList.remove('active');
+
+          thisEl.classList.add('active');
+
+          document.querySelector('.comment_text_js.active').style.display = "none"; // no animation
+
+          for(i=0; i < comments.length; i++) {
+            comments[i].index = i;
+          }
 
-  mobileMenu.addEventListener('click', () => {
-    document.body.classList.toggle('active-menu');
-  })
+          let elIndex = thisEl.index + 1;
 
-  document.addEventListener('click', ev => {
-    const container = document.querySelector('.mob-menu, .mobile_menu_js');
+          document.querySelector('.comment_text_js.active').classList.remove('active');
+          document.querySelector('.comment_text_js:nth-child('+elIndex+'n)').classList.add('active');
+          document.querySelector('.comment_text_js:nth-child('+elIndex+'n)').style.display = "block"; // no animation
```

Comment:
You can use ES6 syntax instead:

```suggestion
          document.querySelector('.comment_text_js.active').classList.remove('active');
          document.querySelector(`.comment_text_js:nth-child('${elIndex}'n)`).classList.add('active');
          document.querySelector(`.comment_text_js:nth-child('${elIndex}'n)`).style.display = "block"; // no animation
```